internal/dockercompose/fake_client.go

```diff
@@ -30,7 +34,28 @@ func (c *FakeDCClient) StreamLogs(ctx context.Context, pathToConfig, serviceName
 }
 
 func (c *FakeDCClient) StreamEvents(ctx context.Context, pathToConfig string) (<-chan string, error) {
-	return nil, nil
+	events := make(chan string, 100)
+	go func() {
```

Comment:
Since this is used in testing I think we should [detect if the buffered channel is full](https://stackoverflow.com/questions/25657207/golang-how-to-know-a-buffered-channel-is-full) and error in that case.